charts/README.md

```diff
@@ -0,0 +1,43 @@
+# The IBM FHIR Server Helm Chart
+[Helm](https://helm.sh/) is a technology for installing software on [Kubernetes](https://kubernetes.io/).
+
+## Prerequisites
+To install the IBM FHIR Server via helm, you must first have a database.
+
+To install a PostgreSQL database to the same cluster using helm, you can run the following command:
+```
+$ helm install my-release bitnami/postgresql
+```
+
+For example if you target the default namespace and name the release "postgres",
+then that would produce something like this:
+```sh
```

Comment:
not clear. where. this ends...

Reply:
it ends on line 43, but I probably need to add some more stuff below there to describe how to actually use the chart.  For now I just wanted to document a way of getting a postgresql in your cluster since I wanted to figure that out anyway